### DISCUSSION TITLE
Make indentation easier to read in yaml files

### DISCUSSION THREAD

**Discussion by Wallman** [community/community#76869] [Repositories]
### Select Topic Area

Product Feedback

### Body

Today, reading and editing yaml files in GitHub is prone to [indentation errors](https://github.com/orgs/community/discussions/25495#discussion-4251810) because of that it's hard to identify the indentation. IntelliJ for example does this a lot better, see attached screenshot.

This applies to both general code viewing as well as code reviewing in PRs.

**GitHub:**
![Screenshot 2023-11-24 at 14 26 34](https://github.com/community/community/assets/4638831/72a41d55-e026-4005-8045-255b08b9c48e)

**IntelliJ:**
![Screenshot 2023-11-24 at 14 25 57](https://github.com/community/community/assets/4638831/f186b9dc-b46c-4a3e-a456-dd396d8b2ad6)

**Comment by jeffstedt** [comment]
Following
